products/appengine/api.yaml

```diff
@@ -502,4 +502,74 @@ objects:
                 Pathname within the host. Must start with a "/". A single "*" can be included at the end of the path.
                 The sum of the lengths of the domain and path may not exceed 100 characters.
               required: true
+  - !ruby/object:Api::Resource
+    name: 'ServiceSplitTraffic'
+    description: |
+      A Service resource is a logical component of an application that can share state and communicate in a secure fashion with other services. 
```

Comment:
"Traffic routing configuration for versions within a single service. Traffic splits define how traffic directed to the service is assigned to versions."